### DISCUSSION TITLE
[Question]: Why is there only arm build for dev and not for non-dev (release)

### DISCUSSION THREAD

**Discussion by iamNoah1** [danny-avila/LibreChat#1938] [Q&A] [answered]
### What is your question?

For the dev version there is a arm image build next to the amd one. But for the non-dev, the release one, the arm build is not done. Is there some specific reason for it? 

### More Details

.

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
I had trouble building an arm build through Github Actions. I only just figured out how to make it work, it's a common issue when dealing with node and arm64 builds: https://github.com/actions/setup-node/issues/462 https://github.com/nodejs/docker-node/issues/1335

When the next release is cut, both architectures will be supported in non-dev images.

**Reply by iamNoah1** [reply]
Cool, thanks for letting me know :)
